src/index.js

```diff
@@ -319,11 +319,18 @@ const initialize = async () => {
      */
 
     sendButton.onclick = async () => {
-      const result = await ethersProvider.getSigner().sendTransaction({
-        to: '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
-        value: '0x29a2241af62c0000',
-        gasLimit: 21000,
-        gasPrice: 20000000000,
+      const result = await ethereum.request({
+        method: 'eth_sendTransaction',
+        params: [
+          {
+            from: accounts[0],
+            to: '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+            value: '3782DACE9D90000',
```

Comment:
Oh, huh, why the change in `value`?

Reply:
previous values of 3 eth was emptying my account 😄

Reply:
This is a most welcome change! Be advised that it may break our e2e tests, but yeah, 3 ETH is too much, and I always end up editing the amount in practice.

One thing, we should `0x`-prefix it for consistency, and if we're changing it, we might as well go even further to 0.1 ETH, as opposed to the current 0.25:

```suggestion
            value: '0x16345785d8a0000',
```

Reply:
@PeterYinusa : please note it may need change in e2e tests for legacy transactions.

Reply:
That's what I was going to say as well - this isn't a valid parameter without the "0x" prefix.

See here: https://eth.wiki/json-rpc/API#hex-value-encoding
This value is a QUANTITY, the rules for which are described there.